frontend/src/scenes/session-recordings/SessionRecordings.tsx

```diff
@@ -136,6 +137,7 @@ export function SessionsRecordings(): JSX.Element {
             />
             <div className="space-y-2">
                 <VersionCheckerBanner />
+                <AndroidRecordingsPromptBanner />
```

Comment:
I wonder how many android users visit the Replay page given that they don't believe it is currently supported 🤔 

Feels like there is a few pages this could make sense on so that it is more likely to be seen:
1. Events
2. Home
3. Replay